invisible_cities/cities/base_cities.py

```diff
@@ -270,6 +270,18 @@ def set_up_database(self):
         self.coeff_blr          = DataPMT.coeff_blr.values      .astype(np.double)
         self.noise_rms          = DataPMT.noise_rms.values      .astype(np.double)
 
+        ## Charge resolution for sensor simulation
+        pmt_single_pe_rms       = DataPMT.Sigma.values .astype(np.double)
+        self.pmt_pe_resolution  = np.divide(pmt_single_pe_rms                         ,
+                                            self.all_pmt_adc_to_pes                   ,
+                                            out=np.zeros_like(self.all_pmt_adc_to_pes),
+                                            where=self.all_pmt_adc_to_pes != 0        )
+        sipm_single_pe_rms      = DataSiPM.Sigma.values.astype(np.double)
+        self.sipm_pe_resolution = np.divide(sipm_single_pe_rms                     ,
+                                            self.sipm_adc_to_pes                   ,
+                                            out=np.zeros_like(self.sipm_adc_to_pes),
+                                            where=self.sipm_adc_to_pes != 0        )
+
```

Comment:
align the `=` in `out` and `where` and surround them with spaces in both `np.divide` calls